lib/sync_checker/formats/edition_base.rb

```diff
@@ -190,6 +192,10 @@ def latest_edition_is_rejected?(document)
           document.pre_publication_edition &&
           document.pre_publication_edition.rejected?
       end
+
+      def disallowed_http_codes
+        (400..599)
```

Comment:
Yep...Personal preference here - it looked odd to me to have a constant range. 

Also, perhaps over thinking the future of the code, but if we wanted to construct custom ranges / arrays of status codes and treat them differently, the method expresses that opportunity, whereas a constant feels more final and as if it closes off code paths. I know it's the same effort to refactor a constant. Like I say, just a feeling. Nothing particularly logical.

I happy to change it though - thoughts?